std/range/package.d

```diff
@@ -10843,14 +10843,26 @@ if (isInputRange!Range && !isInstanceOf!(SortedRange, Range))
     mixin ImplementLength!_input;
 
 /**
-   Releases the controlled range and returns it.
+    Releases the controlled range and returns it.
+
+    This does the opposite of $(LREF assumeSorted): instead of turning a range
+    into a `SortedRange`, it extracts the original range back out of the `SortedRange`
+    using $(REF, move, std,algorithm,mutation).
 */
     auto release()
     {
         import std.algorithm.mutation : move;
         return move(_input);
     }
 
+    ///
+    @safe unittest
+    {
+        auto a = assumeSorted([ 1, 2, 3 ]);
```

Comment:
I think that adding:

```d
assert(a == sort!"a < b"([1, 2, 3]));
```

Might clarify that `a` is a sorted range, whereas release simply extracts the underlying array.